Azure/communication-ui-library-android
# [Feature] Feature/zoom screen share

## Purpose
implemented screen share pinch zoom. on double click the zoom toggles

## Impact
The participant floating header should be displayed as before. (when screen share is on/off)

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
Join group call or teams call with screen share on
Double click on screen share view to make sure on click zoom toggles
Use pinch to scale zoom
Move screen to view some part of screen
Make sure on single click participant header is displayed

## What to Check
Verify that the following are valid
Double click on screen share view to make sure on click zoom toggles
Use pinch to scale zoom
Move screen to view some part of screen
Make sure on single click participant header is displayed
